src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -207,11 +207,21 @@ class HuiEntityButtonCard extends LitElement implements LovelaceCard {
         color: var(--paper-item-icon-color, #44739e);
       }
 
-      ha-icon[data-domain="light"][data-state="on"],
-      ha-icon[data-domain="switch"][data-state="on"],
       ha-icon[data-domain="binary_sensor"][data-state="on"],
+      ha-icon[data-domain="cover"][data-state="open"],
+      ha-icon[data-domain="climate"][data-state="cooling"],
+      ha-icon[data-domain="climate"][data-state="heating"],
+      ha-icon[data-domain="device_tracker"][data-state="home"],
       ha-icon[data-domain="fan"][data-state="on"],
-      ha-icon[data-domain="sun"][data-state="above_horizon"] {
+      ha-icon[data-domain="light"][data-state="on"],
+      ha-icon[data-domain="input_boolean"][data-state="on"],
+      ha-icon[data-domain="lock"][data-state="unlocked"],
```

Comment:
This is the opposite of the active state of a lock.
When locked the lock should have an active icon.